trident/data/line_lists/lines.txt

```diff
@@ -1,5 +1,6 @@
 #Ion        Wavelength [A]           gamma         f_value       alt. name
 H I            1215.670000    4.690000e+08    4.160000e-01            Ly a
+H I            2.1000000e9    2.850000e-15    4.160000e-01            21 cm
```

Comment:
This is quite pedantic, but the rest of the H lines are in order of decreasing wavelength. I would put this line above Lya.